analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -105,6 +106,113 @@
 with open("config.yaml") as config_file:
     config = yaml.safe_load(config_file)
 
+### ML-INFERENCE SETTINGS
+
+# enable ML inference
+USE_INFERENCE = True
+
+# enable inference using NVIDIA Triton server
+USE_TRITON = True
```

Comment:
Should turn this off by default, will just fix in a separate PR as I need to submit another small fix unrelated to the changes in here anyway.